tests/services/test_alerts.py

```diff
@@ -182,88 +182,59 @@ def test_get_details_when_observation_data_not_parseable_remains_unchanged(
         expected_observation_data = '{"invalid_json": ][ }'
         assert observation_data == expected_observation_data
 
-    def test_resolve_when_not_given_tenant_id_posts_expected_data(
+    def test_update_state_when_not_given_tenant_id_posts_expected_data(
         self, mock_connection, user_context, successful_post
     ):
         alert_service = AlertService(mock_connection, user_context)
         alert_ids = ["ALERT_ID_1", "ALERT_ID_2"]
-        alert_service.resolve(alert_ids)
+        alert_service.update_state("RESOLVED", alert_ids, "")
```

Comment:
It would be a slightly better test if you use the non-empty string for the `note` parameter.

Reply:
Added new test.